resources/configurations/configuration.xml

```diff
@@ -104,6 +104,10 @@
             <uri>http://sandbox.rohub.org/rodl/ROs/</uri>
         </roHub>
 
+        <!-- set true if deploying in kubernetes with dns discovery for hazelcast -->
+        <deployInKubernetes>false</deployInKubernetes>
+        <hazelcastDiscoveryK8sDnsService>hazelcast-service.wifire.svc.cluster.local</hazelcastDiscoveryK8sDnsService>
```

Comment:
leave this one blank and change in firemap configuration.xml

Reply:
took these out